README.rst

```diff
@@ -5,3 +5,6 @@ Luftqualität Sachsen
 Eine übersichtliche Darstellung der Messwerte aller sächsischen
 Messstationen zur Luftqualität. Zu jeder Messstation werden die Daten
 detailliert als Tabelle und Diagramm dargestellt.
+
+Installation - Guide
+ >> https://github.com/plumps/luftqualitaet_sachsen/blob/adding-vagrant/docs/installation.rst
```

Comment:
Ich erstelle ein Projekt auf [Read the Docs](https://readthedocs.org/), dann können wir direkt die fertige Doku verlinken.